### ISSUE TITLE
Add Support for more than 4 players

### ISSUE THREAD

**Issue by mmonkey** [meleu/RetroPie-joystick-selection#18]
I do not intend to play with more than four players, however in order to interact with retroarch, you have to have a player bound to each controller you are trying to use. Here is an example:

I have 2 joysticks, 4 SNES and 4 PS3 controllers. The first joystick has 4 extra buttons for dedicated hotkey functions. I would like to be able to use the dedicated buttons on joystick1, while playing a 4 player SNES game.

**Comment by cgmckeever** [comment]
I dont think this is possible based on how retroarch maps input to the hotkeys ... you could enable hotkeys on the console controllers with combos? Thats how its traditionally done, albeit usually painful as I can never remember what clever combo my brain through would be good yesterday

There is also the ability for a keyboard to be the hotkeys ... usually the best UI friendly, but have this extra thing hanging around ..

**Comment by prankard** [comment]
Thanks @meleu for making this enhancement tag.
I've started to work on making this 4 or 8 players. But needs testing and a bug fix. May not fix @mmonkey hotkey issue, but will be cool for multi-tap games
